<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>338</width>
    <height>176</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>338</width>
    <height>176</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>338</width>
    <height>176</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>3</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>3</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <item>
       <widget class="QListWidget" name="list_files"/>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>1</number>
        </property>
        <item>
         <widget class="QPushButton" name="create_steps">
          <property name="text">
           <string>Создать</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="open_steps">
          <property name="text">
           <string>Открыть</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="delete_file_steps">
          <property name="text">
           <string>Удалить</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
